### DISCUSSION TITLE
You need more layers or elements to grow your edition to 10 artworks!

### DISCUSSION THREAD

**Discussion by tonytio** [nftchef/art-engine#186] [Help + Q&A]
DNA exists!
DNA exists!
DNA exists!

I'm getting this error on this test of 'growEditionSizeTo: 10,'

I want to create 2,222 images with json files but i only want to use 1 image with a transparent text layer. So i have 2 folders with one image in each. Backgrounds folder has 1.jpg and Text folder has text.png

Background
{ name: "Background" }
{ name: "Text" },

I have tried naming 1#100.jpg and text#100.jpg but still getting error.

I have tried putting 2,222 images in both Background and Text folders but it's still not working?

Is there a way to do this as I don't need rarities so can I turn this off some how?

Really appreciate any help with how I can achieve this, thanks.

**Comment by BakeSloth** [comment]
you can bypass DNA by adding this to your layerOrder array in config.js

    layersOrder: [
      { name: "Background" 
         options: {
         bypassDNA: true,
       },
      { name: "Text" 
         options: {
         bypassDNA: true,
       },
     ]

**Reply by tonytio** [reply]
Thank you
